src/translations/fr/translation.json

```diff
@@ -1,300 +1,300 @@
     "Prapor Items": "Prapor Items",
-    "Therapist Items": "Therapist Items",
-    "Skier Items": "Skier Items",
-    "Peacekeeper Items": "Peacekeeper Items",
-    "Mechanic Items": "Mechanic Items",
-    "Ragman Items": "Ragman Items",
-    "Jaeger Items": "Jaeger Items",
-    "Tarkov Tools Moobot integration": "Tarkov Tools Moobot integration",
-    "You can add command to your moobot to get price check in your twitch chat": "You can add command to your moobot to get price check in your twitch chat",
-    "using your twitch account": "using your twitch account",
-    "Go to Custom commands": "Go to Custom commands",
-    "Set what you want the command to be. Common is \"p\" or \"price\"": "Set what you want the command to be. Common is \"p\" or \"price\"",
-    "Press the \"Create\" button": "Press the \"Create\" button",
-    "In response choose URL Fetch - Full (plain) response": "In response choose URL Fetch - Full (plain) response",
-    "and after insert \"Command arguments\"": "and after insert \"Command arguments\"",
-    "Now press \"Save\" button": "Now press \"Save\" button",
-    "Ignore settings": "Ignore settings",
-    "Average prices": "Average prices",
-	"Booze Generator": "Booze Generator",
-	"Intelligence Center": "Intelligence Center",
-	"Lavatory": "Lavatory",
-	"Medstation": "Medstation",
-	"Nutrition Unit": "Nutrition Unit",
-	"Water Collector": "Water Collector",
-	"Workbench": "Workbench"
+    "Therapist Items": "Articles de thérapeute",
+    "Skier Items": "Articles de skieur",
+    "Peacekeeper Items": "Articles de maintien de la paix",
+    "Mechanic Items": "Articles de mécanicien",
+    "Ragman Items": "Articles de Ragman",
+    "Jaeger Items": "Articles de jager",
+    "Tarkov Tools Moobot integration": "Intégration des outils Tarkov Moobot",
+    "You can add command to your moobot to get price check in your twitch chat": "Vous pouvez ajouter une commande à votre moobot pour obtenir une vérification des prix dans votre chat Twitch",
+    "using your twitch account": "en utilisant votre compte twitch",
+    "Go to Custom commands": "Aller aux commandes personnalisées",
+    "Set what you want the command to be. Common is \"p\" or \"price\"": "Définissez ce que vous voulez que la commande soit. Commun est \"p\" ou \"prix\"",
+    "Press the \"Create\" button": "Appuyez sur le bouton \"Créer\"",
+    "In response choose URL Fetch - Full (plain) response": "En réponse, choisissez URL Fetch - Réponse complète (plain)",
+    "and after insert \"Command arguments\"": "et après insérer \"arguments de commande\"",
+    "Now press \"Save\" button": "Appuyez maintenant sur le bouton \"Enregistrer\"",
+    "Ignore settings": "Ignorer les réglages",
+    "Average prices": "Prix moyens",
+	"Booze Generator": "Générateur d'alcool",
+	"Intelligence Center": "Centre d'intelligence",
+	"Lavatory": "Toilettes",
+	"Medstation": "Station médicale",
+	"Nutrition Unit": "Unité de Nutrition",
```

Comment:
```suggestion
	"Nutrition Unit": "Ceisine,
```

Reply:
Hey, not sure about this one, let me know what you would like me to do